docs/static/command-line-flags.asciidoc

```diff
@@ -3,81 +3,91 @@
 
 Logstash has the following flags. You can use the `--help` flag to display this information.
 
-[source,shell]
-----------------------------------
--f, --path.config CONFIGFILE
- Load the Logstash config from a specific file, directory, or a wildcard. If
+As an alternative to setting command-line flags, you can specify options in the Logstash settings file,
+`logstash.yml`, located in `LOGSTASH_HOME/config`. Any flags that you set at the command line override the corresponding
+options in the `logstash.yml` file. 
+
+coming[5.0.0-alpha3, Command-line flags have dots instead of dashes in their names]
+
+*`-f, --path.config CONFIGFILE`*::
+ Load the Logstash config from a specific file or directory, or a wildcard. If
  given a directory or wildcard, config files will be read from the directory in
  alphabetical order.
 
--e, --config.string CONFIGSTRING
+*`-e, --config.string CONFIGSTRING`*::
  Use the given string as the configuration data. Same syntax as the config file.
- If not input is specified, 'stdin { type => stdin }' is default. If no output
- is specified, 'stdout { codec => rubydebug }}' is default.
+ If no input is specified, `stdin { type => stdin }` is default. If no output
+ is specified, `stdout { codec => rubydebug }}` is default.
 
--w, --pipeline.workers COUNT
- Sets the number of pipeline workers (threads) to run for filter processing (default: number of cores).
+*`-w, --pipeline.workers COUNT`*::
+ Sets the number of pipeline workers (threads) to run for filter processing (default: half the number of cores).
  If you find that events are backing up, or that the CPU is not saturated, consider increasing
  this number to better utilize machine processing power.
+ 
+*`--pipeline.output.workers COUNT`*::
```

Comment:
IMHO, we should not. Its just inconsistent to have 1 config option being exposed this way. We should not set this precedent.